File: Back/src/main/java/com/example/booking/project/models/entities/AvailabilityModel.java
package com.example.booking.project.models.entities;

import jakarta.persistence.Column;

import java.time.LocalDate;

public class AvailabilityModel {
    @Column
    private LocalDate start_date;
    @Column
    private LocalDate end_date;
    @Column
    private int desired_rooms;
    @Column
    private int desired_guests;
    @Column
    private int min_price;
    @Column
    private int max_price;
}

File: Front/src/app/models/Availability.Model.ts
export class AvailabilityModel {
    start_date: Date;
    end_date: Date;
    desired_rooms: number;
    desired_guests: number;
    min_price: number;
    max_price: number;


    constructor()
    {
        this.start_date = new Date;
        this.end_date = new Date;
        this.desired_rooms = 0;
        this.desired_guests = 0;
        this.min_price = 0;
        this.max_price = 0;
    }
}